bulk_grades/settings/common.py

```diff
@@ -0,0 +1,16 @@
+"""
+Common Pluggable Django App settings
+"""
+
+
+def plugin_settings(settings):
+    """
+    Injects local settings into django settings
+    """
+    settings.ANALYTICS_API_BASE_URL = {
```

Comment:
Why not just one setting with the url and token?

```
settings.ANALYTICS_API_CLIENT = {
        'url': 'http://host.docker.internal:8000/api/v0',
        'token': 'edx'
}
```

Reply:
I followed the same pattern as used in mock proctoring. It could be due to the way prod settings are done.

Reply:
Proctoring allows for multiple backends. That's irrelevant for the analytics API. There's only ever one destination.